lib/nvmf/rdma.c

```diff
@@ -1121,6 +1130,10 @@ nvmf_rdma_qpair_queue_recv_wrs(struct spdk_nvmf_rdma_qpair *rqpair, struct ibv_r
 		rqpair->resources->recvs_to_post.last->next = first;
 		rqpair->resources->recvs_to_post.last = last;
 	}
+
+	if (!rtransport->transport.opts.no_wr_batching) {
```

Comment:
This should be `if (rtransport->transport.opts.no_wr_batching) {` without negotiation

Reply:
Ohh sorry, thanks. I fixed it.